.ci/scripts/release-manager.sh

```diff
@@ -33,7 +35,7 @@ done
 
 # ensure the latest image has been pulled
 IMAGE=docker.elastic.co/infra/release-manager:latest
-docker pull --quiet $IMAGE
+(retry 3 docker pull --quiet "${IMAGE}") || echo "Error pulling ${IMAGE} Docker image, we continue"
```

Comment:
```suggestion
(retry 3 docker pull --quiet "${IMAGE}") || echo "Error pulling ${IMAGE} Docker image, we continue"
docker images --filter=reference=$IMAGE
```

Maybe just list out the image IDs so we know which image we're going to be running in order to help with debugging?